zebra/rt_netlink.c

```diff
@@ -2021,6 +2021,21 @@ static int netlink_neigh_update(int cmd, int ifindex, void *addr, char *lla,
 	if (lla)
 		nl_attr_put(&req.n, sizeof(req), NDA_LLADDR, lla, llalen);
 
+	if (IS_ZEBRA_DEBUG_KERNEL) {
+		char ip_str[256];
```

Comment:
INET6_ADDRSTRLEN + a bit extra for `ipv4/ipv6` keywords would be enough.

Reply:
Thanks for your comments.
Addressed review comments in 0f9a3d765215baf8f0513a0ae2ad8b6157c31b7c